bom/application/pom.xml

```diff
@@ -6,7 +6,6 @@
         <artifactId>quarkus-parent</artifactId>
         <groupId>io.quarkus</groupId>
         <version>999-SNAPSHOT</version>
-        <relativePath>../../pom.xml</relativePath>
```

Comment:
I don't think this does what you think it does.

This will afaik make maven search the parent hierarchy for a a pom matching io.quarkus:quarkus-parent:999-SNAPSHOT before looking in repositories.

That might in effect be the same but it does risk taking time when importing projects in IDE's etc.

I think what you want is to put <relativePath/> to short-circuit this and make it explicit to not search relatively first.

Reply:
`<relativeParent/>` should be put everywhere we use an explicit parent.